bin/cloudgov/files-pull

```diff
@@ -1,4 +1,23 @@
-#!/bin/sh
+#!/bin/bash
+
+usage="
+$0: Pulls media files from s3.
```

Comment:
Add "This script pulls the files required to populate s3/local/cms/public on a developer's filesystem."